.ci/env/apt.sh

```diff
@@ -54,35 +54,80 @@ function install_dev-base-conda {
     conda env create -f .ci/env/environment.yml
 }
 
-function install_arm-cross-compilers {
-    sudo apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu gfortran-aarch64-linux-gnu
+function install_gnu-cross-compilers {
+    sudo apt-get install -y "gcc-$1-linux-gnu" "g++-$1-linux-gnu" "gfortran-$1-linux-gnu"
 }
 
-function install_qemu_emulation {
+function install_qemu_emulation_apt {
     sudo apt-get install -y qemu-user-static
 }
 
+function install_qemu_emulation_deb {
+    sudo apt-get install -y binfmt-support
+    qemu_deb=qemu-user-static_8.2.2+ds-2+b1_amd64.deb
+    wget http://ftp.de.debian.org/debian/pool/main/q/qemu/${qemu_deb}
+    sudo dpkg -i ${qemu_deb}
+    sudo update-binfmts --importdir "$1" --import "qemu-$2"
```

Comment:
That's is already done as part of the dpkg -i above

Reply:
Local testing didn't update the binfmt. I tried installing the .deb package using `apt-get install` to the same effect

Reply:
Aha. Thanks Mark. That does work for me